mesos-client/.scalafmt.conf

```diff
@@ -0,0 +1,3 @@
+maxColumn = 120
+includeCurlyBraceInSelectChains = false
+includeNoParensInSelectChains = false
```

Comment:
Should this be in all `.scalafmt.conf`? What do you think about symlink all to just one file?

Reply:
I'll add support to the plugin to accept absolute paths and then we'll be able to do just one config. That shouldn't be that much work so this will go

Reply:
But when is that done? Why not simply do the symlink for now?

Reply:
@alenkacz I understand that the plugin will be updated soon... and sounds good!   however can we use the same file everywhere.    The top level project has 2 lines... this one has 3 lines.   we should have the same definition in all places.